### PR TITLE
Dev/documentation

### PR DESCRIPTION
**Pull request by vcanogil**
Updating docs. 

This should result in having a simple outline of the purpose of the library and an example of using it.

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -5,20 +5,20 @@ DocMeta.setdocmeta!(Cclib, :DocTestSetup, :(using Cclib); recursive=true)
 
 makedocs(;
     modules=[Cclib],
-    authors="Felipe S. S. Schneider <schneider.felipe@posgrad.ufsc.br> and contributors",
```
Comment (shivupa) [inline docs/make.jl:8]: `authors="cclib development team",`

**Review by shivupa — CHANGES_REQUESTED** [review]
1 minor change

File: docs/make.jl (hunk 1 above)
Comment (vcanogil) [reply docs/make.jl:8]: Done

**Review by berquist — APPROVED** [review]
Looks good, but how do I build the docs from the command line?

**Pull request merged**